binancial/data/get_trades_historical.py

```
@@ -1,4 +1,4 @@
-def get_trades_historical(client, symbol="BTCUSDT"):
+def get_trades_historical(client, symbol="BTCUSDT", limit=1000):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_trades_historical`

(ANN201)